src/index.js

```diff
@@ -319,5 +357,56 @@ function populateCustomerDash(customerDashInfo) {
   bookingsList.innerHTML = generateBookingsList(customerDashInfo.userBookings); 
 }
 
-// it should also be used to calculate & display their total spent(functions already exist for this ?)
-//   it should also be used to generate a list of their bookings(again, functions already exist for this; may need to be made more dynamic)
+function getRoomsAvailableOnDate() {
+  event.preventDefault();
+  let dateSelected = getDateSelected(); 
+  const roomsBooked = getRoomNumbersOnDate(dateSelected); 
+  return roomRepo.getAvailableRooms(roomsBooked);
+}
+
+function getDateSelected() {
+  let dateSelected = document.getElementById('customer-search').value;
```

Comment:
Currently, it does not let you use the calendar drop down

Reply:
Thanks, I'll add this to the backlog to look into. Seems this problem is on both my manager & customer dash